cooltools/api/rearrange.py

```diff
@@ -178,7 +181,7 @@ def reorder_cooler(
     if new_chrom_col not in view_df.columns:
```

Comment:
- throw a Warning if new_chrom_col name is not provided, but the default value is taken. 
- not sure about randint, but may be ok (this is an unlikely scenario anyway)

Reply:
Added the warning, but actually this can only happen when the user explicitly provides new_chrom_col=None or orientation_col=None, since the arguments have default values.